client/src/locales/en-US.json

```diff
@@ -1625,7 +1625,28 @@
                 "description": "You can change your credit card at any time.",
                 "update": "Update"
             },
-            "processing": "Your order is being processed."
+            "processing": {
+                "requestSent": {
+                    "title": "Request sent",
+                    "description": "Your request is being processed. We will contact you within 4 working days."
```

Comment:
Where is this "4 days" coming from, we don't know how fast the sales team will treat the requests, remove this and replace by something like 'as soon as possible'..